neon_homeassistant_skill/__init__.py

```diff
@@ -1,12 +1,13 @@
+# pylint: disable=missing-function-docstring,missing-class-docstring,missing-module-docstring
 import asyncio
 from typing import List
+
+from mycroft import intent_handler
```

Comment:
I think this has a reference in `ovos_workshop` now that should help with dependency warnings (and tests in the future)